README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
```

Comment:
It would be useful to include instructions on how to verify that 'go-task' was installed successfully, to help users troubleshoot.